tests/structures/test_exception.py

```diff
@@ -63,3 +63,21 @@ def test_repr(self):
             print('>>> print(repr(ValueError([1], 1, \\'1\\')))')
             print(repr(ValueError([1], 1, '1')))
         """)
+
+    def test_str(self):
+        self.assertCodeExecution("""
+            print('>>> print(str(ValueError()))')
+            print(repr(ValueError('')))
```

Comment:
Shouldn't this be `print(str(ValueError()))`?

Reply:
Sorry. I wasn't clear that there were two changes I was suggesting, `repr` to `str` and `('')` to `()`.